rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -385,6 +393,7 @@ function createRuleTable(gridId, data, needCheckbox, needActions, needCompliance
 
   // Choose which columns should be included
   var columns = [];
+  columns.push(tags);
```

Comment:
If you had pushed them at the end you would'nt have to change the filter column ;)  but since it's done, it's ok!!